schedule.md

```diff
@@ -111,21 +110,22 @@ The following schedule is subject to change, and many slide links are not yet ac
 </div>
 <div class="week">
 
-## Week 5 (October 25-27)
+## Week 5 (October 30-November 1)
 
 ### Topics
+
 - ### Tuesday October 15: Zoom Guest Lecture with Alexander Pucher
 - Relational Databases and Rails ([slides](/slides/2022/11_relational_databases))
 - RDBMS Scaling ([slides](/slides/2022/12_rdbms_scaling))
 
 </div>
 <div class="week">
 
-## Week 6 (November 1-3)
+## Week 6 (November 6-8)
 
 ### Topics
 
-### Thursday November 3: Guest Lecture with David Acevedo
+### Wednesday November 8: Guest Lecture with David Acevedo
 
 - Deep Dive: Load Testing with Tsung ([slides](/slides/2022/10_tsung))
 - Visiting [North Hall Data Center](https://www.it.ucsb.edu/enterprise-technology-services/north-hall-data-center)
```

Comment:
This will need to be arranged ahead of time if you want to organize a visit. Let's DM on this.